### PR TITLE
HUD changes

### PR DESCRIPTION
**Pull request by rubenwardy**
It may be worth explaining the types of elements whilst doing the example, but I need to get around to doing that. `lua_api.txt` should be for listing a particular HUD's properties, however the book should probably explain things in a better way

0,0 being upper left should be stated

### CODE REVIEW

File: en/chapters/hud.md [hunk 1]
```diff
@@ -10,83 +10,101 @@ Heads Up Display (HUD) elements allow you to show text, images, and other graphi
...
+a percentage position and an offset.
+The percentage position is relative to the screen size, so to place an element
+in the center of the screen you would need to provide a percentage position of half
+the screen, eg (50%, 50%), and an offset of (0, 0).
+
+<figure>
+    <img
+        width="300"
+        src="{{ page.root }}/static/hud_diagram_center.svg"
+        alt="The player inventory formspec, with annotated list names.">
```
Comment (rubenwardy) [inline en/chapters/hud.md]: oops

**Comment by rubenwardy** [comment]
I suggest:

* A section on placing the player's HUD items in the correct way (eg: using hud_add)
    * placing text items
    * placing background
* A section on storing the player's HUD IDs
    * a table to store them, create function
* A section on modifying the player's HUD info
   * update function

The example form will look like:

```
---------------------------------
|  Dug nodes                 35 |
|  Placed nodes              35 |
---------------------------------
```

4 text elements with different colors and alignments, and an image background

**Comment by Ezhh** [comment]
This sounds good to me. Giving a clear process with ordered steps, as well as a practical example, should help a lot on this page.

**Comment by rubenwardy** [comment]
https://rubenwardy.com/hud/

**Pull request merged**